source/ui.js

```diff
@@ -87,7 +87,7 @@ module.exports = async (options, pkg) => {
 					return 'Please specify a valid semver, for example, `1.2.3`. See http://semver.org';
 				}
 
-				if (!version.isVersionGreater(oldVersion, input)) {
+				if (!version.isVersionGreaterThan(oldVersion, input)) {
```

Comment:
Since the import name is `version`, couldn't we remove the word `version` from the methods?
```suggestion
				if (!version.isGreaterThan(oldVersion, input)) {
```

Reply:
Actually, we could make it even more readable by making it `version('0.1.0').isGreaterThan('0.0.1')`. Thoughts?

Reply:
> Actually, we could make it even more readable by making it `version('0.1.0').isGreaterThan('0.0.1')`. Thoughts?

I agree that it's more readable. Note that we'll still have separate methods which work without a given version (for now, only `isValidVersionInput()`, though that might change in the future).